baselines/dom.generated.d.ts

```diff
@@ -3701,9 +3701,9 @@ declare var Event: {
 }
 
 interface EventTarget {
-    addEventListener(type: string, listener?: EventListenerOrEventListenerObject, useCapture?: boolean): void;
+    addEventListener(type: string, listener?: EventListenerOrEventListenerObject, useCapture?: boolean | AddEventListenerOptions): void;
```

Comment:
According to the spec, the third argument should be named "options" instead of `useCapture`, can you also correct this?